### ISSUE TITLE
Post Thumbnails Not Generated in SB 2.7

### ISSUE THREAD

**Issue by karks88** [WebDevStudios/StartBox#41]
I've been using 2.7 in a test site (using WP 3.5.1 Multisite) and I've found that the post thumbnails that are supposed to show up on the category archive don't appear to be there.  For example, I uploaded a featured image for a post, and the source code is looking for an image at:

http://xxxx.xxxx.com/fia/usa/wp-content/uploads/sites/2/2013/03/vanuatu1-200x200-tc.jpg

The file is not found on the server.

**Comment by brichards** [comment]
Could you provide us with real working URLs so we can test your implementation by chance? It's hard to follow imaginary links to see what's going on :)

**Comment by karks88** [comment]
Sure, here's the main URL:
http://ekwds77.pairserver.com/fia/

As I said, this is multi-site.  But only the landing page and USA section are really setup at this point.

You can see the example of post thumbnails not generating at:
http://ekwds77.pairserver.com/fia/usa/category/projects/

Pay particular attention to the "Vanuatu: Base Development and Tribal Ministry" article, as that is the one with a featured image assigned.

**Comment by brichards** [comment]
Hm, I think there might be something up with your network setup... http://demo.wpstartbox.com/ is a 3.5.1 network, running SB 2.7, with no issues on the thumbnails.

Can you tell me more about your setup? I notice we're a few subdirectories deep here.

**Comment by karks88** [comment]
Well, it's supposed to consist of 3 sites in a network:

-Landing/Intro page (main directory)
-USA (usa directory)
-Canada (canada directory)

Right now I'm working on the USA site. 

It's running on Pair Networks hosting with FreeBSD 8.2, Apache 2.2.23, MySQL 5.1.61 and PHP 5.3.18.  I hope this helps!

**Comment by karks88** [comment]
I'm starting to wonder if it has anything to do with the fact that I host with Pair Networks?  Just installed WP 3.5.1 with SB 2.7.2 and I get the following error when I have post thumbnails enabled:

function.getimagesize]: http:// wrapper is disabled in the server configuration by allow_url_fopen=0 in /usr/home/paawwa/public_html/paawwa.org/wp-content/themes/startbox/includes/functions/images.php on line 170

Warning: getimagesize(http://66.39.35.196/wp-content/themes/startbox/images/nophoto.jpg) [function.getimagesize]: failed to open stream: no suitable wrapper could be found in /usr/home/paawwa/public_html/paawwa.org/wp-content/themes/startbox/includes/functions/images.php on line 170

The URL of the install is 
http://66.39.35.196/

EDIT: Setting allow_url_fopen to yes will fix the issue. Not sure if this is just a problem with Pair Networks or if any other hosts with allow_url_fopen to no will cause the problem.
